addons/website_blog/views/snippets/s_blog_posts.xml

```diff
@@ -183,7 +199,7 @@
 <!-- Single record templates -->
 <!-- Full Layout -->
 <template id="dynamic_filter_template_blog_post_single_full" name="Solo Full">
-    <t t-if="len(records)" class="s_blog_posts_post" data-extra-snippet-classes="o_cc o_cc5" data-column-classes="col-lg-12">
+    <t t-if="len(records)">
         <t t-set="data" t-value="records[0]"/>
         <t t-set="record" t-value="data['_record']"/>
```

Comment:
You didn't adapt this one and the other ones below to be like the ones above (so with the data-attributes as comments) :open_mouth: They are needed to render the snippets correctly :slightly_smiling_face: 
e.g. with `dynamic_filter_template_event_event_single_aside` (in the `s_events.xml` file) since there isn't the `data-column-classes=12` attribute anymore, this Event snippet layout is broken (because it doesn't take 12 columns anymore): 
- With the class:
  <img width="1389" height="808" alt="image" src="https://github.com/user-attachments/assets/0f36ebba-88ae-4dd4-8cf5-766d1c5be4c9" />
- Without the class (so in your branch):
  
<img width="1919" height="706" alt="image" src="https://github.com/user-attachments/assets/e389a4c9-2cb3-41ce-a667-f8e2b5bed62a" />